packages/vue-instantsearch/src/__tests__/common-widgets.test.js

```diff
@@ -362,6 +362,45 @@ const testSetups = {
       document.body.appendChild(document.createElement('div'))
     );
 
+    await nextTick();
+  },
+  async createCurrentRefinementsWidgetTests({
+    instantSearchOptions,
+    widgetParams,
+    testParams,
+  }) {
+    const refinementListAttributes = Object.keys(
+      instantSearchOptions.initialUiState?.indexName.refinementList || {}
+    );
+
+    const search = (h) =>
+      h(AisInstantSearch, { props: instantSearchOptions }, [
+        ...refinementListAttributes.map((attribute) =>
+          h(AisRefinementList, { props: { attribute } })
```

Comment:
we're no longer testing the output of current refinements with widgets other than refinement list

Reply:
Good call, I'll put a test with the full suite of facets / refinements in the common test suite.

Reply:
yes, could be in the shared widget tests

Reply:
Done in https://github.com/algolia/instantsearch/pull/5838/commits/3a1c4a480cf93a1a4d0f44a62a2981f4eefa0d62. Although, I haven't found a way to add `tag` and `exclude` refinement types without access to the helper.

Reply:
do the tag and exclude refinement tests still exist in the connector? if so that's good enough imo